Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml

```diff
@@ -7,6 +7,7 @@
     Flash: 5
     FlashlightSeclite: 5
     ClothingEyesGlassesSecurity: 4
+    ClothingEyesGlassesSecurityPrescription: 2
```

Comment:
Move this to the bottom. Just for the sake of organization (and merge conflicts), it's nice to have anything we add at the bottom of a list.